source/features/bugs-tab.tsx

```diff
@@ -29,7 +35,7 @@ const countBugs = cache.function(async (): Promise<number> => {
 		search(type: ISSUE, query: "label:bug is:open is:issue repo:${getRepo()!.nameWithOwner}") {
 			issueCount
 		}
-	`);
+	`) as Search;
```

Comment:
Can we not?

Reply:
K I had this from last lint and I forgot to commit it. I will revert.